web_src/js/features/repo-home.js

```diff
@@ -140,8 +153,11 @@ export function initRepoTopicBar() {
     },
     onLabelCreate(value) {
       value = value.toLowerCase().trim();
-      this.attr('data-value', value).contents().first().replaceWith(value);
-      return $(this);
+      // `this` is the default label (jQuery object), it's a `$('<a class="ui small label">')`
+      // we create a new div element to replace it, to keep the same as template (repo-topic-label), because we do not want the `<a>` tag which affects aria focus.
+      const $el = $(`<div class="ui small label topic gt-cursor-default" data-value="${htmlEscape(value)}">${htmlEscape(value)}${svg('octicon-x', 16, 'delete icon gt-ml-3 gt-mt-1')}</div>`);
```

Comment:
Is there no better way than duplicating alle the classes here which will eventually go out of sync?

Reply:
If you want to make it a general approach to use SVG (instead of writing customized code for every dropdown), the `templates.label` must be delegated - like the `templates.menu` in #23542 , or just make it global by `$.fn.dropdown.templates.label = ...`

https://github.com/fomantic/Fomantic-UI/blob/980ea5e4f4c63a5d549f175f9974420bc4e07bb7/dist/components/dropdown.js#L4314-L4321


And then you need to delegate the `onLabelCreate` to fine tune the `label` itself, Fomantic already hard-coded it to be a `<a>`

https://github.com/fomantic/Fomantic-UI/blob/980ea5e4f4c63a5d549f175f9974420bc4e07bb7/src/definitions/modules/dropdown.js#L2839-L2856

Reply:
ps: I will discard this PR after #23542 (if it could be approved), and start a clear one.